pages/common/musescore.md

```diff
@@ -0,0 +1,32 @@
+# musescore
+
+> MuseScore 3 sheet music editor.
+> More information: <https://musescore.org/en/handbook/3/command-line-options>.
+
+- Use a specific audio driver:
+
+`musescore --audio-driver {{jack|alsa|portaudio|pulse}}`
+
+- Set the MP3 output bitrate in kbit/s:
+
+`musescore --bitrate {{bitrate}}`
+
+- Start MuseScore in debug mode:
+
+`musescore --debug`
+
+- Enable experimental features, such as layers:
+
+`musescore --experimental`
+
+- Export the given (or currently opened) file to the specified output file. The file type depends on the extension given:
+
+`musescore --export-to {{output_file}} {{input_file}}`
+
+- Print a conditioned `diff` between the given scores:
```

Comment:
What do you mean by 'conditioned diff'?

Reply:
`--diff` prints it in a more readable way in contrast to `--raw-diff`. Should I just delete that word or do you have a better alterantive?

Reply:
I would just removed 'conditioned'.  I don't think it matters much to get every detail in.  Also, I think you remove backticks from 'diff' because, it's not even that much of a technical word, much less a command.